### PR TITLE
Document elasticsearch backup and restore via s3.

### PR DESCRIPTION
**Pull request by jmcarp**
I'm going to run through this again and double-check for accuracy, but I think it's close enough for review.

### CODE REVIEW

File: content/docs/services/elasticsearch24.md [hunk 1]
```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
...
 pre-ElasticSearch 2.0 but was disabled in versions 2.0 to 2.3. Learn more about
 that [here](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/dots-in-names.html).
 
 ### Rotating credentials
 
 You can rotate credentials by creating a new instance and deleting the existing instance. If this is not an option, email [cloud.gov support](mailto:cloud-gov-support@gsa.gov) to request rotating the credentials manually.
 
+### Managing backups
+
+The Elasticsearch service includes the [AWS Cloud Plugin](https://www.elastic.co/guide/en/elasticsearch/plugins/2.4/cloud-aws.html), which supports snapshot and restore with AWS S3. For detailed instructions, see the [Snapshot and Restore](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/modules-snapshots.html) [Cloud AWS Repository](https://www.elastic.co/guide/en/elasticsearch/plugins/2.2/cloud-aws-repository.html) documentation. You can also use this simple example. The example assumes you already have an elasticsearch service called `my-elasticsearch` and an app called `my-app`.
```
Comment (LinuxBozo) [inline content/docs/services/elasticsearch24.md]: Might want to mention you need`jq` and `curl` before trying the rest.

File: content/docs/services/elasticsearch24.md [hunk 2]
```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
...
+    cf create-service s3 basic my-s3-bucket
+    ```
+
+* Create a [service key](https://docs.cloudfoundry.org/devguide/services/service-keys.html) to access S3 credentials:
+
+    ```sh
+    cf create-service-key my-elasticsearch-backup my-key
+    ```
+
+* Connect to your elasticsearch service using port forwarding. Note: You'll need to leave the `cf ssh` command running and follow the next steps in a different terminal:
```
Comment (LinuxBozo) [inline content/docs/services/elasticsearch24.md]: I would mention we need to leave ssh running specifically for the localized port forwarding.

File: content/docs/services/elasticsearch24.md [hunk 3]
```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
...
+
+    ```sh
+    s3_credentials=$(cf service-key my-s3-bucket my-key)
+
+    s3_bucket=$(echo "${s3_credentials}" | jq -r '.bucket')
+    s3_region=$(echo "${s3_credentials}" | jq -r '.region')
+    s3_access_key=$(echo "${s3_credentials}" | jq -r '.access_key_id')
+    s3_secret_key=$(echo "${s3_credentials}" | jq -r '.secret_key_id')
+
+    curl -X PUT -u "${es_username}:${es_password}" "${es_hostname}:9200/_snapshot/my_s3_repository" -d @<(cat <<EOF
```
Comment (LinuxBozo) [inline content/docs/services/elasticsearch24.md]: Should this be localhost, not {es_hostname}?

File: content/docs/services/elasticsearch24.md [hunk 4]
```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
...
+      }
+    }
+    EOF
+    )
+    ```
+
+* Create a snapshot:
+
+    ```sh
+    curl -X PUT -u "${es_username}:${es_password}" "${es_hostname}:9200/_snapshot/my_s3_repository/my_s3_snapshot"
```
Comment (LinuxBozo) [inline content/docs/services/elasticsearch24.md]: Same thing here, should be localhost yeah?

File: content/docs/services/elasticsearch24.md [hunk 5]
```diff
@@ -31,14 +31,79 @@ cf create-service elasticsearch24 1x my-elastic-service
...
+    ```
+
+* Restore from a snapshot. Note: to restore to a new elasticsearch instance, you'll need to connect to your new instance (see "Connect to your elasticsearch service using port forwarding" above) and add the existing S3 repository (see "Create a snapshot repository" above).
+
+    ```sh
+    curl -X PUT -u "${es_username}:${es_password}" "${es_hostname}:9200/_snapshot/my_s3_repository/my_s3_snapshot/_restore"
```
Comment (LinuxBozo) [inline content/docs/services/elasticsearch24.md]: ... and again, localhost?

**Comment by jmcarp** [comment]
Updated, WDYT @LinuxBozo ?

**Review by LinuxBozo — APPROVED** [review]

**Pull request merged**